### PR TITLE
Jhardaway 484 dyamic blog description

### PR DESCRIPTION
**Pull request by jeromehardaway**
### Summary
📝🚀🛠️

This pull request updates the frontmatter and SEO components of several blog posts in the `src/data/blogs` and `src/pages/blogs` directories. The changes add author names, descriptions, and images to the blog posts, and use them to dynamically generate the SEO attributes of the blog post pages. The purpose of these changes is to improve the readability, SEO, and accessibility of the blog posts on the Vets Who Code app.

> _We write the code of the brave and the bold_
> _We add the fields to the frontmatter of doom_
> _We update the types and the SEO_
> _We make the posts shine like a full moon_

### Walkthrough
*  Added author name and description fields to the frontmatter of three blog posts and updated the image source of one blog post to improve readability, SEO, and accessibility ([link](https://github.com/Vets-Who-Code/vets-who-code-app/pull/503/files?diff=unified&w=0#diff-abd90a1955ff4f917350b25de386f47e77c3f5637b58de8b22e229772a224807L4-R8), [link](https://github.com/Vets-Who-Code/vets-who-code-app/pull/503/files?diff=unified&w=0#diff-b4360e71db44c1058760fe88f0e54418505138d7a45ed5225429ed596b4d2707L3-R4), [link](https://github.com/Vets-Who-Code/vets-who-code-app/pull/503/files?diff=unified&w=0#diff-7a8823be498d1d42dc912a0c17606dc2ce9310b2f239f1a58cd8b2e450ca8e88L4-R5), )
*  Changed the `is_featured` field of one blog post to `true` to mark it as a featured article on the homepage ([link](https://github.com/Vets-Who-Code/vets-who-code-app/pull/503/files?diff=unified&w=0#diff-b4360e71db44c1058760fe88f0e54418505138d7a45ed5225429ed596b4d2707L19-R19))
*  Changed the `description` and `image` props of the `SEO` component in `src/pages/blogs/[slug].tsx` to dynamically render the description and image of each blog post based on the frontmatter data ([link](https://github.com/Vets-Who-Code/vets-who-code-app/pull/503/files?diff=unified&w=0#diff-07fa73f864faad0fbb6a8bd785cc56c67c0acd0397bdfd5acae38b10d95da2dcL42-R42), [link](https://github.com/Vets-Who-Code/vets-who-code-app/pull/503/files?diff=unified&w=0#diff-07fa73f864faad0fbb6a8bd785cc56c67c0acd0397bdfd5acae38b10d95da2dcL50-R50))
*  Added the `description` field to the interface `BlogMetaType` in `src/utils/types.ts` to update the type definition of the blog post metadata ([link](https://github.com/Vets-Who-Code/vets-who-code-app/pull/503/files?diff=unified&w=0#diff-98a58367e6df376c04f7d50445cf8b648654b7533801565e9f215037c219e5edR177))

### CODE REVIEW

**Review by ayumi-b — APPROVED** [review]

File: src/utils/types.ts [hunk 1]
```diff
@@ -174,6 +174,7 @@ export interface BlogMetaType {
 }
 
 export interface IBlog {
+    description: string | undefined;
```
Comment (gixxerblade) [inline src/utils/types.ts:177]: Make optional

Comment (mrbernnz) [reply src/utils/types.ts:177]: Do you mean change the syntax?

```
- description: string | undefined
+ description?: string
```

Comment (jeromehardaway) [reply src/utils/types.ts:177]: @gixxerblade, it's already optional. If you delete the JSON section that would house the description, it will not break.

Comment (jeromehardaway) [reply src/utils/types.ts:177]: @mrbernnz I didn't, I will change that in another branch or if you could check out the branch and change it I will approve it.

**Pull request merged**